Dockerfile-ci

```diff
@@ -0,0 +1,17 @@
+FROM ubuntu
+MAINTAINER Michael Bartling "michael.bartling15@gmail.com"
+
+RUN apt-get update
+RUN apt-get install -y \
+    git \
+    python3 \
+    python3-pip \
+    gcc \
+    wget \
+    lcov \
+    g++ 
+RUN pip3 install --upgrade pip setuptools wheel
+COPY requirements.txt .
+RUN pip3 install -r requirements.txt
```

Comment:
I thinks as for CI, it's better to install the package with `python3 setup.py install`.

Reply:
So we can check if it will works on `PyPI`.

Reply:
Fixed